### ISSUE TITLE
Power Socket

### ISSUE THREAD

**Issue by TheAmazingSpy** [Electrical-Age/ElectricalAge#331]
I think this is a great idea, and also would go great with a whole bunch of other cables. So this outlet can be put onto any block, and then on the back of that block, you can now wire it. These will only take 50v before exploding and the cables, I was thinking something like Immersive Engineering's type, with the cable being like string and that, and you have an item called 'Cable' that you right click on the outlet and then you hook it up to another machine. Then there are 2 sockets on the outlet, and if you right click the left with an empty hand, it toggles it on and off, and same with the other, except the right. If you want to get really technical, add in a expander board, but make it that with enough expander boards in the cheap tier, it will catch fire, but if you get a surge proof one it will just turn off

**Comment by TheAmazingSpy** [comment]
Alright, the socket: 
![screen shot 2015-12-06 at 6 47 34 pm](https://cloud.githubusercontent.com/assets/13247665/11612334/f07db212-9c49-11e5-8a76-e9add0aa3440.png)

**Comment by TheAmazingSpy** [comment]
The orange sticks are levers

**Comment by Domi1993** [comment]
What do you want to connect there? A Tableau with cicruitbreakers like at your home will needed for it or are the orange levers a circuitbreaker?

**Comment by TheAmazingSpy** [comment]
No, they are switches on the power socket to toggle on and off. And this would use be for 50 volt things

**Comment by lambdaShade** [comment]
Power socket models are already done since the commit https://github.com/Dolu1990/ElectricalAge/commit/9f49e13de044c6b08863c274e6b431f1d6f31eac. These will be used with the new distribution board (https://github.com/Dolu1990/ElectricalAge/commit/c8a42bb53e61c6598737cbca2ab6d51524d4c1b8).
These changes are planned for the next release. ;-)

**Comment by cm0x4D** [comment]
Switzerland connector :-D

**Comment by Domi1993** [comment]
Bisch du schwiizer? Are you swiss?

**Comment by lambdaShade** [comment]
;-)
The 50V socket is effectively inspired on the type J socket (Swiss SEV 1011) and the 200V one on the type E/F.

**Comment by cm0x4D** [comment]
@Domi1993: Yes, actually quite a lot of us lives in Valais.

@lambdaShade: I actually like the model very much, but a rectangular hole might match the Minecraft style a little better. But that is just a proposal, it is already fine as it is.

**Comment by Domi1993** [comment]
I'm swiss too from Canton Zug

**Comment by lambdaShade** [comment]
Yeah... I should probably "rectangularize" the 200V version. The 50V is historical :-)
In low priority todo list !

**Comment by jrddunbr** [comment]
We did this in LL, and can be ported easily.
